be/src/vec/exec/volap_scan_node.cpp

```diff
@@ -1876,7 +1870,6 @@ int VOlapScanNode::_start_scanner_thread_task(RuntimeState* state, int block_per
     }
 
     // post volap scanners to thread-pool
-    auto cur_span = opentelemetry::trace::Tracer::GetCurrentSpan();
```

Comment:
Why remove this?

Reply:
This span is passed into the scanner thread, and a span is collected in each scanner thread, which may cause too many spans to be generated